### DISCUSSION TITLE
あなたの好きな料理を教えてください

### DISCUSSION THREAD

**Discussion by HK-TO** [NAIST-SE/AttractingContributors#455] [General]
あなたの好きな料理を教えてください．
Please tell me your favorite cooking.

https://github.com/HK-TO/AttractingContributors-Cooking

**Comment by ghost** [comment]
Hi! my favorite cooking is karaage!
If you have time, please write something to this repository over here
(https://github.com/MakotoSato0321/AttractingContributors-Anime/tree/main).
